src/parser/test/ParserTest.cpp

```diff
@@ -594,28 +607,36 @@ TEST(Parser, InsertEdge) {
 TEST(Parser, UpdateEdge) {
     {
         GQLParser parser;
-        std::string query = "UPDATE EDGE 12345 -> 54321 SET amount=3.14,time=1537408527";
+        std::string query = "UPDATE EDGE 12345 -> 54321 OF transfer "
```

Comment:
UPDATE EDGE `edge name` will better ?

Reply:
I can't understand your idea, can you elaborate on it? thx.